tracer/src/Datadog.Trace/DatabaseMonitoring/DatabaseMonitoringPropagator.cs

```diff
@@ -0,0 +1,52 @@
+// <copyright file="DatabaseMonitoringPropagator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Propagators;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.DatabaseMonitoring
+{
+    internal static class DatabaseMonitoringPropagator
+    {
+        private const string SqlCommentSpanService = "dddbs";
+        private const string SqlCommentRootService = "ddps";
+        private const string SqlCommentVersion = "ddpv";
+        private const string SqlCommentEnv = "dde";
+
+        internal static string PropagateSpanData(DbmPropagationLevel propagationStyle, string configuredServiceName, Span span)
```

Comment:
With the other changes I suggested below, it looks like you can pass a `SpanContext` instead of `Span`, which is preferable. The `SpanContext` (and its associated `TraceContext`) contains everything you need for propagation, and is easier to test.